e2e-tests/functions

```diff
@@ -255,6 +255,7 @@ wait_cluster_consistency() {
 	sleep 7 # wait for two reconcile loops ;)  3 sec x 2 times + 1 sec = 7 seconds
 	until [[ "$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.mysql.state}')" == "ready" &&
 	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.mysql.ready}')" == "${cluster_size}" &&
+	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.orchestrator.ready}')" == "${orc_size}" &&
 	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.orchestrator.state}')" == "ready" ]]; do
```

Comment:
Why is state value not enough here? (Just for my better understanding (= )

Reply:
Because status can be wrong, eg. we might have a bug where status is Ready, but the number of ready pods is 2/3 or something like that. In automatic tests it is always good to put more check points, because automatic is not like human where when you see something you will see that it doesn't make sense. In automation if you don't put a specific check it will just ignore everything else even if it doesn't make any sense.